tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -3646,26 +3648,47 @@ public function userSendsPropfindRequestToSpaceWithHeaders(string $user, string
 	 * @param string $spaceName
 	 * @param string|null $resource
 	 * @param array|null $headers
+	 * @param string|null $folderDepth
 	 *
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 *
 	 * @throws JsonException
 	 */
-	public function sendPropfindRequestToSpace(string $user, string $spaceName, ?string $resource = "", ?array $headers = []): ResponseInterface {
+	public function sendPropfindRequestToSpace(string $user, string $spaceName, ?string $resource = "", ?array $headers = [], ?string $folderDepth = "0"): ResponseInterface {
```

Comment:
`1` should be default value
```suggestion
	public function sendPropfindRequestToSpace(string $user, string $spaceName, ?string $resource = "", ?array $headers = [], ?string $folderDepth = "1"): ResponseInterface {
```